src/api/demo.rkt

```diff
@@ -416,6 +417,31 @@
      (define repr (context-repr (test-context test)))
      (hasheq 'points (pcontext->json pctx repr) 'job id 'path (make-path id)))))
 
+(define explanations-endpoint
+  (post-with-json-response (lambda (post-data)
+                             (define formula
+                               (read-syntax 'web (open-input-string (hash-ref post-data 'formula))))
+                             (define sample (hash-ref post-data 'sample))
+                             (define seed (hash-ref post-data 'seed #f))
+                             (eprintf "Explanations job started on ~a..." formula)
```

Comment:
Nit pick: Adding a `\n` newline character after `...` to match the print out of the other test.

Reply:
Also right now this is printing `formula` results in the following output. `#<syntax:web::1 (FPCore (x) (- (sqrt (+ x 1)) (sqrt x)))>` The `#<syntax:web...` can be removed by printing `(hash-ref post-data 'formula)` instead.